examples/open_loop_control_fetch/run.py

```diff
@@ -31,43 +32,79 @@
+    elif args.rendering_mode == "raytracing":
+        rendering_mode_map_str = "_rtx"
+        config.defrost()
+        config.SIMULATION_CONTROLLER.CAMERA_SENSOR.FINAL_COLOR_INDIRECT_LIGHTING_INTENSITY = 0.0
+        config.freeze()
+    else:
+        assert False
+
+    # change config based on current scene
+    config.defrost()
+
+    if args.scene_id == "starter_content_0000":
+        # starter_content_0000 doesn't need a rendering mode when referring to its map
+        config.SIMULATION_CONTROLLER.SCENE_ID = args.scene_id
+        config.SIMULATION_CONTROLLER.MAP_ID   = args.scene_id
+
+        # starter_content_0000 has scene-specific config values
+        scene_config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "scene_config.starter_content_0000.yaml")
+
+    elif args.scene_id == "kujiale_0000":
+        config.SIMULATION_CONTROLLER.SCENE_ID = args.scene_id
+        config.SIMULATION_CONTROLLER.MAP_ID   = args.scene_id + rendering_mode_map_str
+
+        # kujiale_0000 has scene-specific config values
+        scene_config_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "scene_config.kujiale_0000.yaml")
+
+    config.merge_from_file(scene_config_file)
+    config.freeze()
 
     # create Env object
     env = spear.Env(config)
 
-    # reset the simulation to get the first observation    
+    # reset the simulation to get the first observation
     obs = env.reset()
 
     if args.benchmark:
         start_time_seconds = time.time()
     else:
-        cv2.imshow("camera.final_color", obs["camera.final_color"])  # note that spear.Env returns BGRA by default
-        cv2.waitKey(1)
+        if args.save_images:
+            if os.path.exists(args.image_dir):
+                shutil.rmtree(args.image_dir)
+            os.makedirs(args.image_dir)
 
-    for i,row in df.iterrows():
-        action = { k:np.array([v], dtype=np.float32) for k,v in row.to_dict().items() }
+    for i, row in df.iterrows():
+        action = {k: np.array([v], dtype=np.float32) for k, v in row.to_dict().items()}
         obs, reward, done, info = env.step(action=action)
 
-        if not args.benchmark:
-            cv2.imshow("camera.final_color", obs["camera.final_color"]) # note that spear.Env returns BGRA by default
-            cv2.waitKey(1)
+        # save images for each render pass
+        if not args.benchmark and args.save_images:
+            for render_pass in config.SIMULATION_CONTROLLER.CAMERA_AGENT.CAMERA.RENDER_PASSES:
+                render_pass_dir = os.path.realpath(os.path.join(args.images_dir, render_pass))
+                assert os.path.exists(render_pass_dir)
 
-        if args.save_images:
-            cv2.imwrite(os.path.realpath(os.path.join(args.image_dir, f"{i:04d}.jpg")), obs["camera.final_color"])
+                obs_render_pass = obs["camera." + render_pass].squeeze()
+                if render_pass in ["final_color", "lens_distortion", "normals", "segmentation"]:
```

Comment:
let's remove "lens_distortion" for consistency with our other examples, and because I'd like to remove it from CameraSensor soon, so I don't want people to start using it

Reply:
```
                obs_render_pass = obs["camera." + render_pass].squeeze()
                if render_pass in ["final_color", "normals", "segmentation"]:
                    assert len(obs_render_pass.shape) == 3
                    assert obs_render_pass.shape[2] == 4
                    obs_render_pass = obs_render_pass[:, :, [2, 1, 0, 3]].copy()  # note that spear.Env returns BGRA by default
```

Reply:
fixed